applications/portfolioDrafts/portfolio/createPortfolioStep.ts

```diff
@@ -20,9 +21,28 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
     );
   }
 
-  const portfolioId = event.pathParameters?.portfolioDraftId;
-
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+  if (!portfolioDraftId) {
+    return new ErrorResponse(
+      { code: ErrorCodes.INVALID_INPUT, message: "PortfolioDraftId must be specified in the URL path" },
+      ErrorStatusCode.BAD_REQUEST
+    );
+  }
+  if (!IsValidJson(event.body)) {
```

Comment:
Could we make it so that we had a

```typescript
const requestBody = parse(event.body);
if (!requestBody) {
  // return Invalid JSON response
}
```
?